clients/web/src/utilities/MiscFunctions.js

```diff
@@ -36,24 +36,22 @@ girder.formatDate = function (datestr, resolution) {
 };
 
 /**
- * Format a size in bytes into a human-readable string with metric unit prefixes.
+ * Format a size in bytes into a human-readable string with metric unit
+ * prefixes.
  */
 girder.formatSize = function (sizeBytes) {
-    // If it's > 1GB, report to two decimal places, otherwise just one.
-    var precision = sizeBytes > 1073741824 ? 2 : 1;
```

Comment:
I see the point.

I think that I'd rather see a minimum number of digits of significance (say 4), so that until you cross the 100 Gb threshold, you'd have two decimal places.  That format looks poor with small values, since 2345 bytes ends up being '2.290 kB', so then you need further logic not to apply the reduction to values below 10000.